deep-learning/_posts/2017-09-06-semi_predDepth.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: review
+title: "Semi_Supervis for  Depth Prediction"
```

Comment:
The title of the paper is
>Semi-Supervised Deep Learning for Monocular Depth Map Prediction

I would suggest to shorten this to: **Semi-Supervised Monocular Depth Map Prediction**